prefab/curl-ssl/README.md

```diff
@@ -1,12 +1,27 @@
 # curl-ssl
 
-The curl-ssl sample shows how to import native dependencies from Maven. This
-sample uses [curl], [OpenSSL], and [JsonCpp] to display a list of the most
-recent 10 reviews submitted to AOSP's code review system.
+The curl-ssl sample shows how to use [Prefab] to import native dependencies from [Google Maven]. This  
```

Comment:
I'd revert the change to this line. Prefab is an implementation detail that's not important, and gmaven is just one of the repositories this works with,

Reply:
done